cmd/podman/machine/config.go

```diff
@@ -0,0 +1,38 @@
+package machine
+
+import "fmt"
+
+type CreateOptions struct {
+	CPUS       uint64
+	Memory     uint64
+	KernelPath string
+	Devices    []VMDevices
+}
+
+type VMDevices struct {
+	Path     string
+	ReadOnly bool
+}
+
+type VM interface {
+	Create(name string, opts CreateOptions) error
+	Start(name string) error
+	Stop(name string) error
+}
+
+type TestVM struct {
```

Comment:
Does the test code here still need to exist?

Reply:
Nope, this is residual from when I was testing interfaces, so it should be removed.

Reply:
removed.